### ISSUE TITLE
Deployment on Azure.

### ISSUE THREAD

**Issue by JimBobSquarePants** [Taritsyn/JavaScriptEngineSwitcher#11]
Would it be possible for `JavaScript Engine Switcher for .Net: V8` to wrap a release build of ClearScript so that the extra C++ binaries would not be required? 

This would make it much easier to deploy on Azure as well as other servers.
https://msdn.microsoft.com/en-us/library/azure/hh694038.aspx?f=255&MSPPError=-2147217396

At present I am having to use `JavaScript Engine Switcher for .Net: Jint` in my project [Cruncher](https://github.com/JimBobSquarePants/Cruncher) due to the difficulties in deployment which is non optimal as Jint takes an extraordinarily long time to initialise and parser on first run.

**Comment by Daniel15** [comment]
What's wrong with extra C++ binaries? The link you posted seems to say that it's fine as long as you deploy the .dll files correctly. Could you use the MSIE engine in Azure instead? The Azure VMs may support that natively.

**Comment by Taritsyn** [comment]
> Would it be possible for JavaScript Engine Switcher for .Net: V8 to wrap a release build of ClearScript so that the extra C++ binaries would not be required? 

What's stopping you to add the `msvcp110.dll` and `msvcr110.dll` assemblies in your application package? I think it is unnecessary to add the standard assemblies in the heavy JavaScriptEngineSwitcher.V8 package.

> At present I am having to use JavaScript Engine Switcher for .Net: Jint in my project Cruncher due to the difficulties in deployment which is non optimal as Jint takes an extraordinarily long time to initialise and parser on first run.

Jint and Jurassic originally are slow JS engines, because it are written on managed code.

> ... in my project [Cruncher](https://github.com/JimBobSquarePants/Cruncher) ...

In [CoffeeScript](https://github.com/JimBobSquarePants/Cruncher/tree/master/Cruncher/Preprocessors/Coffee) and [Autoprefixer](https://github.com/JimBobSquarePants/Cruncher/tree/master/Cruncher/Postprocessors/AutoPrefixer) functionality I saw a lot of familiar code from the [Bundle Transformer](https://bundletransformer.codeplex.com/) :-)

In general, tools such as the Bundle Transformer, [Cassette](http://getcassette.net/) and [SquishIt](https://github.com/jetheredge/SquishIt) are gradually fading. Now need switch to [Grunt](http://gruntjs.com/) or [Gulp](http://gulpjs.com/). In [“Additional reading and resources”](https://bundletransformer.codeplex.com/wikipage?title=Additional%20reading%20and%20resources) section of the Bundle Transformer documentation, there is information on theme of switching to Grunt/Gulp.

**Comment by Daniel15** [comment]
> In general, tools such as the Bundle Transformer, Cassette and SquishIt are gradually fading. Now need switch to Grunt or Gulp.

Also the ASP.NET 5 beta does not include ASP.NET Web Optimization Framework (System.Web.Optimization) at all, instead it uses Grunt.

**Comment by Daniel15** [comment]
Having said that I think there's still value in .NET tools for combination/minification, in scenarios where you don't want to use Node.js tools like Grunt/Gulp/Webpack.

**Comment by Taritsyn** [comment]
I just voiced recommendations of Microsoft.

**Comment by JimBobSquarePants** [comment]
Well... It is some fantastically clever code! :smile: 

I made a point of saying where I found it in the source also of course...

@Daniel15  As I recall the MSIE engine didn't work on Azure, I'll check again though. 
@Taritsyn, I'm actually surprised that Microsoft have gone that direction, Node is a nightmare on Windows machines and adding another dependency can be a hassle.

I'll probably just add the files in my solution. Thanks.
